testdata/snapshots/TestDemo0.testza

```diff
@@ -0,0 +1 @@
+(string) ""
```

Comment:
This is an empty snapshot, which is probably committed by accident?

Reply:
Good catch, there was a `setup` call missing. Demo snapshots were working when run directly, but not with other tests.